### ISSUE TITLE
Support for Wayland

### ISSUE THREAD

**Issue by Faalagorn** [marcincichocki/breach-protocol-autosolver#367]
**Is your feature request related to a problem? Please describe.**

Currently, on Linux, automatic input support is handled by `xdotool` which doesn't work for Wayland-based compositors, like sway or KDE or GNOME Wayland.

**Describe the solution you'd like**

Support for some X11-neutral tool, like `ydotool` or `dotool` or similar. 

**Describe alternatives you've considered**

Some tool other than `ydotool` or `dotool` entirely. 

**Additional context**

Wayland is moving forward, slowly replacing X11, so it'd be nice to have support for it on Linux.

**Comment by cgarz** [comment]
Yes please support wayland, it really is moving forward quickly now. Arch's KDE just recently updated to plasma 6 and now that I've got around to updating this no longer works and having to manually solve them again is so painful :grimacing:.

**Comment by jtmackoy** [comment]
For those still waiting for Wayland support - a workaround is to use the "Analyze from File" feature ...

1. Use your favorite Wayland-friendly screen cap tool to take an image of the puzzle.
2. Click the drop-down arrow in the upper-right corner of the autosolver window and select `Analyze` -> `From file`.
3. Select the file created from step 1.
4. Manually solve the puzzle using one of the solutions the autosolver suggests.
5. Quite literally, at least in-game ... profit.
